packages/ui/src/examples/ResponsiveCard/ResponsiveCardExample.tsx

```diff
@@ -0,0 +1,45 @@
+import { Divider, Flex, Heading, Image, Stack, Text } from '@chakra-ui/react'
+
+export const ResponsiveCardExample = () => {
+  return (
+    <Flex
+      flexDirection={['column', null, 'row']}
+      gap={3}
+      bg="white"
+      m={3}
+      p={5}
+    >
+      <Stack>
+        <Image
+          w={['350px', null, 'full']}
+          h={['350px', null, 'full']}
+          src="https://picsum.photos/600/400"
+          borderRadius={['full', null, 'none']}
+          m="auto"
+        ></Image>
+      </Stack>
+      <Stack spacing={7} w={['full', null]} p={3}>
```

Comment:
```suggestion
      <Stack spacing={6} w={['full', null]} p={3}>
```

In most cases I use even numbers (2, 4, 8, 16) for spacings (margin, padding, gap etc).